.github/workflows/continuous-integration-lint-php.yml

```diff
@@ -44,7 +44,11 @@ jobs:
         run: vendor/bin/phpcs -q --report=checkstyle --runtime-set ignore_errors_on_exit 1 --runtime-set ignore_warnings_on_exit 1 | cs2pr --graceful-warnings
 
       - name: Static Analysis (PHPStan)
-        run: vendor/bin/phpstan analyse --memory-limit=512M --error-format=checkstyle | cs2pr
+        run: composer phpstan -- --error-format=checkstyle | cs2pr
+        if: always()
+
+      - name: Static Analysis (PHPMD)
+        run: composer phpmd
```

Comment:
Shame that this can not be output checkstyle / cs2pr

Reply:
I know right!? 😞